docs/advanced-customization/config/newsletter.md

```diff
@@ -1,11 +1,11 @@
 ---
 description: How to configure Mailchimp newsletters with your Forem
-position: 12
+position: 13
 ---
 
 # Newsletter
 
-Newsletter settings allow you to integrate your Mailchimp newsletter audience with your Forem community members. Newsletters are sent from your Mailchimp account and are different from your [Forem Email Digests](https://admin.forem.com/docs/advanced-customization/config/email-digest-frequency).
+Newsletter settings allow you to integrate your Mailchimp newsletter audience with your Forem community members. Newsletters are sent from your Mailchimp account and are different from your [Forem Email Digests](https://admin.forem.com/docs/advanced-customization/config/emails#periodic-email-digest).
```

Comment:
I've updated all links for the digest email and email server settings to point to the correct updated pages